ddtrace/_monkey.py

```diff
@@ -352,9 +352,10 @@ def _patch_all(**patch_modules: bool) -> None:
 
     patch(raise_errors=False, **modules)
     if asm_config._iast_enabled:
-        from ddtrace.appsec._iast._patch_modules import patch_iast
         from ddtrace.appsec.iast import enable_iast_propagation
 
+        from .appsec._iast.main import patch_iast
```

Comment:
can we avoid non absolute path in import? It makes static analysis more difficult.

Reply:
```suggestion
        from ddtrace.appsec._iast.main import patch_iast
```